include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -195,19 +198,20 @@ class CommonOperatorLoader : public ProtobufLoader {
     NodeValue in = getNodeValueOrCreateVariableByName(op.input(0));
 
     // Get the requested shape from the model.
-    // First look at input tensors, then at the "shape" attribute.
+    // First look for the "shape" argument, otherwise search for an input tensor
+    // with the shape.
     std::vector<int64_t> requestedDims;
-    if (op.input_size() > 1) {
+    if (dict.count("shape")) {
+      std::vector<int64_t> protoDims = getShape<int64_t>(dict["shape"]);
```

Comment:
Why isn't it a mistake to have both `shape` and `op.input(1)` present?

Reply:
Ah, this was actually occurring only because of my hand-hacked proto for seq2seq. You're right, it's an error. I replaced that commit with one that asserts it doesn't happen.